src/pages/home/layouts/layout-liquidity-pool.tsx

```diff
@@ -28,9 +39,54 @@ export const LiquidityPoolLayout = () => {
   // TODO: pool 구성에 있는 토큰 리스트
   const tokens = [{ symbol: 'MOAI' }, { symbol: 'XRP' }, { symbol: 'ROOT' }, { symbol: 'WETH' }];
 
+  const handleTokenClick = (token: string) => {
+    if (selectedTokens.includes(token)) {
+      setSelectedTokens(selectedTokens.filter(t => t !== token));
+    } else {
+      setSelectedTokens([...selectedTokens, token]);
+    }
+  };
+
+  const sortTokensBySelection = (tokens, selectedTokens: string[]) => {
+    return tokens.sort((a, b) => {
+      const isASelected = selectedTokens.includes(a.symbol);
+      const isBSelected = selectedTokens.includes(b.symbol);
+
+      if (isASelected && !isBSelected) {
+        return -1;
+      } else if (!isASelected && isBSelected) {
+        return 1;
+      } else {
+        return 0;
+      }
```

Comment:
p4; early return 을 애용합시당
```suggestion
      if (isASelected && !isBSelected) return -1
      if (!isASelected && isBSelected) return 1
      return 0
```

Reply:
[fix: early return in sort token func, add annotation at isMounted che…](https://github.com/TeamHeimdallr/moai-web/pull/146/commits/dbe84ac4b06603e259b5b50375a51675f01b37bc)